sympy/core/expr.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function, division
```

Comment:
We don't need future imports for py3 things as python 2 isn't supported any more.